elks/arch/i86/kernel/irqtab.S

```diff
@@ -392,7 +149,7 @@ ktask:				// Already using interrupt stack
 //
 //	Already using interrupt stack, keep using it
 //
-	mov	%sp,%si
+	//mov	%sp,%si
```

Comment:
Might as well delete this since its' commented out and loaded above.

If you want to also delete the various other commented out segment prefixes from the BCC conversion years ago:
// seg cs (or es etc), you might as well do that too.

Reply:
I kept that MOV instruction commented to hightlight the fact it is performed on routine entry just above and is no more necessary at that place.

Reply:
OK - but far more than that is required for others to understand this routine. At this point, a cleaner diff resulting from commit will be easier to read without it, IMO.

Reply:
Okay, that change without any impact would be performed in a next PR.